README.md

```diff
@@ -32,9 +32,11 @@ The following must be available in your system:
 - [yarn](https://yarnpkg.com/)
 - [python](https://www.python.org/)
 - [pipenv](https://pypi.org/project/pipenv/)
-- [urllib3](https://urllib3.readthedocs.io/en/stable/) without this the project will not run in ```macOS``` environments
+- [urllib3](https://urllib3.readthedocs.io/en/stable/) without this the project will not run in `macOS` environments
```

Comment:
Those are cli requirements, we should mention that one must install the cli and its requirements and just link to crowdbotics/cli repo